ui/src/app/dashboard/grouped-summary/grouped-summary.component.css

```diff
@@ -1,14 +1,60 @@
-.mat-card {
+#container {
+  display: inline-block;
+  width: 25rem;
   margin: 1rem;
 }
 
-table a {
+div.expanded mat-card {
+  padding-bottom: 0.2rem;
+}
+
+div.collapsed mat-card {
+  padding-bottom: 0.2rem;
+}
+
+div.collapsed clr-icon.arrow {
+  transform: rotate(180deg);
+}
+
+div.expanded clr-icon.arrow {
+  transform: rotate(0deg);
+}
+
+table {
+  width: 100%;
+}
+
+a {
   text-decoration: none;
   color: #000000;
 }
 
-.mat-card-title {
-  font-size: 1.2rem;
-  color: gray
+tr > th.label {
+  width: 50%;
+  height: 30px;
+  font-size: large;
 }
 
+tr > td.label-value {
+  width: 40%;
+}
+
+div.button-container {
+  text-align: center;
+}
+
+
```

Comment:
It would be nice to have a clearer visual distinction between the following things:
- labels
- counts
- 0-value counts

Perhaps for now just a slightly lighter font color for the numbers?